src/Processors/AugmentProperties.php

```diff
@@ -113,4 +112,26 @@ public function __invoke(Analysis $analysis)
             }
         }
     }
+
+    /**
+     * @param string $type
+     *
+     * @return string
+     */
+    protected function exractType(string $typeString)
```

Comment:
why $typeString when the variable already has a type declarations. just name it $type. you can also add a return type

Reply:
Because in foreach I also use types as $type and I don't want to confuse the reader with a var named the same but with different content. With typeStrign I mean "The string which contains the type in it but also holds more info". I will think about a better naming.

Reply:
I simply reused the given naming. ;)